spec/request/person_spec.rb

```diff
@@ -8,14 +8,14 @@
     post "/users/login", params: {email: @u.email, password: @u.password}
   end
 
-  it "should let you edit your profile" do
-    patch "/admin/people/#{@p.id}", params: { person: {given_name: "New Name"}}
+  it "should let you edit your profile, using session cookie to determine who you are" do
```

Comment:
these tests were failing after a refactor from last week, now we use session cookies to determine who a user is (and therefore whose profile to edit)